frontend/app-development/hooks/mutations/useRepoPushMutation.ts

```diff
@@ -2,7 +2,7 @@ import { useMutation, useQueryClient } from '@tanstack/react-query';
 import { useServicesContext } from '../../common/ServiceContext';
 import { QueryKey } from '../../types/QueryKey';
 
-export const useRepoPushMutation = (owner, app) => {
+export const useRepoPushMutation = (owner, app, setShouldShowSpinner) => {
```

Comment:
Now that this parameter is passed in, we can use it in the `onSuccess`-function on l.10.
So on f.ex. l.11, you can add
```javascript
setShouldShowSpinner(false);
```

Reply:
added